packages/component-model-manuscript/entities/manuscript/index.test.js

```diff
@@ -1,11 +1,18 @@
-const { createTables } = require('@pubsweet/db-manager')
+const { createTables } = require('@elifesciences/component-model')
 const uuid = require('uuid')
 const Team = require('@elifesciences/component-model-team').model
 const User = require('@elifesciences/component-model-user').model
 const File = require('@elifesciences/component-model-file').model
 const AuditLog = require('@elifesciences/component-model-audit-log').model
 const Manuscript = require('.')
 
+let logs = []
```

Comment:
may be worth putting in a comment so we know to remove once tests are stable again.

Reply:
I removed this logging as its only in one test and can easily be re-added if need be